### PR TITLE
Banner updates for regular primary page

### LINKED ISSUE #2298: Banner Update: Replace green/blue banner
Followup from https://github.com/mozilla/foundation.mozilla.org/issues/1727 and https://github.com/mozilla/foundation.mozilla.org/issues/1721 and comps https://redpen.io/p/avd5fc0b3db14a7883

### Replace green/blue banner and add call out box

Needed on primary pages like Internet Health, People (although I think this page will be archived soon), About Us, News, Fellowships. Not needed on special primary pages like Home, Initiatives, Participate, or other pages like Campaign and Opportunity pages.

- [x] Swap out the old graphic (gradient and white line art) for one of the new b&w patterns (black/white marble header in https://foundation.mozilla.org/cms/images/)
- [x] Add white cutout box
- [x] Move title off of image and into box (update to h1-heading)
- [x] Add intro text (large-body)

Note: right now banner is all the same but we might make it different on each page https://github.com/mozilla/foundation.mozilla.org/issues/1663

###  Desktop

screen shot 2018-12-18 at 10 08 02 am

### Mobile

screen shot 2018-12-18 at 10 07 43 am

### Overview
nav new 2x

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2298 

Notes: copy for intro paragraph is to be entered on CMS

- https://foundation-mofostaging-pr-2423.herokuapp.com/about/
- https://foundation-mofostaging-pr-2423.herokuapp.com/people/
- https://foundation-mofostaging-pr-2423.herokuapp.com/fellowships/
- https://foundation-mofostaging-pr-2423.herokuapp.com/styleguide/

### CODE REVIEW

File: source/sass/global.scss [hunk 1]
```diff
@@ -6,31 +6,6 @@ body {
     background: $white;
   }
 
-  // Subpages
```
Comment (mmmavis) [inline source/sass/global.scss]: Renamed the vague selector name from `.wrapper-2` to `#hero` and cleaned up DOM a bit so SCSS portion got updated too.

**Comment by kristinashu** [comment]
Yay looking good! But sorry, I think some of the styles don't quite reflect @beccaklam latest changes.

- [ ] title should be h1 instead of h5
- [ ] white box should span 8 columns instead of 3
- [ ] add a little more margin (big squares are 48px)

Screen cap of comp with guides:
image

Screen cap of PR:
image

**Comment by kristinashu** [comment]
Also what to clarify that this banner will also be on [Awards](http://foundation.mozilla.org/awards/), [IH](https://foundation.mozilla.org/en/internet-health/), and [News](https://foundation.mozilla.org/en/news/).

**Comment by mmmavis** [comment]
PR updated!

@kristinashu 

> Also what to clarify that this banner will also be on Awards, IH, and News.

Yes changes in this PR will reflect on the pages you mentioned above as they are of page type "Primary Page". I have also filed a ticket to make sure review apps include these pages by default https://github.com/mozilla/foundation.mozilla.org/issues/2427

**Comment by mmmavis** [comment]
yay!

**Pull request merged**